ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/extensions/PaywallDataExtensions.kt

```diff
@@ -0,0 +1,99 @@
+package com.revenuecat.purchases.ui.revenuecatui.extensions
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+import androidx.core.os.LocaleListCompat
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.paywalls.PaywallColor
+import com.revenuecat.purchases.paywalls.PaywallData
+import java.net.URL
+
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.Companion.createDefault(packages: List<Package>): PaywallData {
+    return PaywallData.createDefault(packages.map { it.identifier })
+}
+
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.Companion.createDefault(packageIdentifiers: List<String>): PaywallData {
+    val locale = LocaleListCompat.getDefault()[0].toString()
+
+    return PaywallData(
+        templateName = PaywallData.defaultTemplate,
+        config = PaywallData.Configuration(
+            packages = packageIdentifiers,
+            images = PaywallData.Configuration.Images(
+                background = PaywallData.defaultBackgroundImage,
+                icon = PaywallData.defaultAppIconPlaceholder,
+            ),
+            colors = PaywallData.defaultColors,
+            blurredBackgroundImage = true,
+            displayRestorePurchases = true,
+        ),
+        localization = mapOf(locale to PaywallData.defaultLocalization), // TODO-PAYWALLS: test this
+        assetBaseURL = PaywallData.defaultTemplateBaseURL, // TODO-PAYWALLS: we need to load a resource not a URL
+        revision = PaywallData.revisionID,
+    )
+}
+
+// region Internal defaults
+
+internal val PaywallData.Companion.defaultTemplate: String // TODO-PAYWALLS: use enum
```

Comment:
Probably many of these can be private. And maybe we could even just put them into an `object` like:
```
object DefaultPaywallConstants {
   ....
}
```
To avoid having so many extensions

Reply:
I actually think extensions are simpler  to maintain because they indicate what type they're relevant for (`PaywallData`) in this case, as opposed to having a completely different type with a random name.

Reply:
I can do something like this too to make the constants file private

```
private const val defaultAppIconPlaceholder = "revenuecatui_default_paywall_app_icon"
```

WDYT?

Reply:
Well, my only concern there is having a lot of things declared at the top level... But honestly, I'm ok either way, that would also work for me.